dawn/main/MenuTemplate/DebugMenu.js

```diff
@@ -2,8 +2,8 @@
  * Defines the debug menu.
  */
 
-import RendererBridge from '../RendererBridge';
 import { fork } from 'child_process';
+import RendererBridge from '../RendererBridge';
```

Comment:
Linter caught this; there's a [rule](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/first.md) in there to specify absolutes before relatives.